src/index.ts

```diff
@@ -1,6 +1,4 @@
-// content type validator
-export { contentTypeValidator } from './lib/client/ContentTypeValidator';
```

Comment:
`contentTypeValidator` a été renommé `validateContentType`, en soi why not, mais dans ce cas, il faut prévoir :
- de livrer une version majeure
- un guide de migration

Reply:
ha oui j'avais pas pensé à ça .. si je l'exporte aussi avec son ancien nom ca passe ?

Reply:
ouais, à vérifier que ça marche, mais sur le principe oui !